src/literals.py

```diff
@@ -10,7 +10,7 @@
 
 from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, StatusBase, WaitingStatus
 
-CHARMED_ZOOKEEPER_SNAP_REVISION = 30
+CHARMED_ZOOKEEPER_SNAP_REVISION = 33
```

Comment:
**todo** can we just update this with the revision published officially? I have pinged @taurus-forever to update the credentials for the charmed-zookeeper-snap repo, such that we can publish the snap properly. Once that is done, we can update this PR with the correct revision

Reply:
@Batalex I believe Alex has updated the credentials, retriggered the CI and a new snap has been published ([reference](https://github.com/canonical/charmed-zookeeper-snap/actions/runs/8797498685/job/24192011975)). Thanks @taurus-forever